pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -23,7 +23,7 @@ const (
 	//
 	// Maintainers: Run ./Documentation/check-crd-compat-table.sh for each release
 	// Developers: Bump patch for each change in the CRD schema.
-	CustomResourceDefinitionSchemaVersion = "1.24.2"
+	CustomResourceDefinitionSchemaVersion = "1.24.3"
```

Comment:
Do we have an automated script to bump these versions?

Reply:
Yeah, it's part of the release process, `Documentation/check-crd-compat-table.sh`. On this commit for instance it complained, presumably because something in the CRD changed between the previous release and now. So we'll bump the schema version here and then Cilium should automatically bump the CRD schema upon upgrade.

Reply:
For reference, here's the change:

```
diff --git a/pkg/k8s/apis/cilium.io/client/crds/v2/ciliumnodes.yaml b/pkg/k8s/apis/cilium.io/client/crds/v2/ciliumnodes.yaml
index cf756759b23e..9a44141219b0 100644
--- a/pkg/k8s/apis/cilium.io/client/crds/v2/ciliumnodes.yaml
+++ b/pkg/k8s/apis/cilium.io/client/crds/v2/ciliumnodes.yaml
@@ -519,6 +519,23 @@ spec:
                         description: Error is the error message set by cilium-operator.
                         type: string
                     type: object
+                  release-ips:
+                    additionalProperties:
+                      description: IPReleaseStatus  defines the valid states in IP
+                        release handshake
+                      enum:
+                      - marked-for-release
+                      - ready-for-release
+                      - do-not-release
+                      - released
+                      type: string
+                    description: 'ReleaseIPs tracks the state for every IP considered
+                      for release. value can be one of the following string : * marked-for-release
+                      : Set by operator as possible candidate for IP * ready-for-release  :
+                      Acknowledged as safe to release by agent * do-not-release     :
+                      IP already in use / not owned by the node. Set by agent * released           :
+                      IP successfully released. Set by operator'
+                    type: object
                   used:
                     additionalProperties:
                       description: AllocationIP is an IP which is available for allocation,
```